public/app/core/components/Login/LoginPage.tsx

```diff
@@ -37,28 +39,41 @@ export const LoginPage: FC = () => {
             changePassword,
             skipPasswordChange,
             isChangingPassword,
+            resetCode,
           }) => (
             <div className={loginStyles.loginOuterBox}>
-              {!isChangingPassword && (
+              {!(isChangingPassword || forgotPassword || resetCode) && (
```

Comment:
If there is something more added here, it definitely has to be rewritten to some kind of routing solution.